pkg/apis/environment/basic_view.go

```diff
@@ -65,9 +68,46 @@ type (
 		runtime.RequestCollection[
 			predicate.Environment, environment.OrderOption,
 		] `query:",inline"`
+
+		IncludeSummary bool `query:"includeSummary,omitempty"`
 	}
 
-	CollectionGetResponse = []*model.EnvironmentOutput
+	CollectionGetResponse = []*environmentOutput
 )
 
 type CollectionDeleteRequest = model.EnvironmentDeleteInputs
+
+type environmentOutput struct {
+	model.EnvironmentOutput `json:",inline"`
+	StatusSummary           status.Count `json:"statusSummary"`
+}
+
+func exposeEnvironment(entity *model.Environment) *environmentOutput {
+	output := &environmentOutput{
+		EnvironmentOutput: *model.ExposeEnvironment(entity),
+	}
+
+	if len(entity.Edges.Resources) > 0 {
+		for _, v := range entity.Edges.Resources {
+			switch {
+			case v.Status.Error:
+				output.StatusSummary.Error++
+			case v.Status.Transitioning:
+				output.StatusSummary.Transitioning++
+			default:
+				output.StatusSummary.Ready++
```

Comment:
A stopped resource is neither in error nor transitioning status, is it expected to count it as ready?

Reply:
The stopped resource count as "ready" is confusing. What do you think about changing "ready" to "healthy" or "stable"?

Reply:
Ready, Error and Other?